test/e2e/post-911-gib-status-helpers.js

```diff
@@ -0,0 +1,89 @@
+const mock = require('./mock-helpers');
+
+const enrollmentData = {
```

Comment:
Saw the intermediate commit where you were looking for specific data values. Maybe something in between is just to look for a single data-dependent item in each section?